crates/forge/tests/integration/prank.rs

```diff
@@ -177,3 +177,67 @@ fn start_prank_in_constructor_test() {
 
     assert_passed!(result);
 }
+
+#[test]
+fn stop_prank() {
+    let test = test_case!(
+        indoc!(
+            r#"
+            use result::ResultTrait;
+            use array::ArrayTrait;
+            use option::OptionTrait;
+            use traits::TryInto;
+            use starknet::ContractAddress;
+            use starknet::Felt252TryIntoContractAddress;
+            use cheatcodes::PreparedContract;
+
+            #[starknet::interface]
+            trait IPrankChecker<TContractState> {
+                fn get_caller_address(ref self: TContractState) -> felt252;
+            }
+
+            #[test]
+            fn test_stop_prank() {
+                let class_hash = declare('PrankChecker');
+                let prepared = PreparedContract { class_hash, constructor_calldata: @ArrayTrait::new() };
+                let contract_address = deploy(prepared).unwrap();
+                let contract_address: ContractAddress = contract_address.try_into().unwrap();
+                let dispatcher = IPrankCheckerDispatcher { contract_address };
+            
+                let target_caller_address: felt252 = 123;
+                let target_caller_address: ContractAddress = target_caller_address.try_into().unwrap();
+
+                let old_caller_address = dispatcher.get_caller_address();
+
+                start_prank(target_caller_address, contract_address);
```

Comment:
Please add test cases for all of the cheatcodes:
- Calling on a not pranked contract
- Prank contract twice then stop

Reply:
Calling on a not pranked contract is already tested afaik. I think I'd better to add double pranking in #336 instead

Reply:
> I think I'd better to add double pranking in https://github.com/foundry-rs/starknet-foundry/issues/336 instead

seems unrelated to the PR you mentioned, can you create an issue for it so we do not miss it?